packages/cli/src/utils.ts

```diff
@@ -133,10 +132,33 @@ const checkImageExtension = (uri: string): boolean => {
   );
 };
 
-export const showUserErrorMessage = (msg: string) => {
-  console.error("\n:::: Solana Publish CLI: Error Message ::::");
-  console.error(msg);
-  console.error("");
+export const generateNetworkSuffix = (rpcUrl: string): string => {
+  let suffix = "";
+
+  if (rpcUrl.indexOf("devnet") != -1)
+    suffix = "?cluster=devnet"
+  else if (rpcUrl.indexOf("testnet") != -1)
+    suffix = "?cluster=testnet"
+  else if (rpcUrl.indexOf("mainnet") != -1)
+    suffix = "?cluster=mainnet"
```

Comment:
we need an auto-formatter setup; i swear `prettier` was enabled. i'll _actually_ block on this... can you run `pnpm fmt` on all this? these "if else" ladders without curly braces gives me the heebie-jeebies

Reply:
Hah hah, I don't like them either, I just thought in Typescript land it'd be more okay since it's more "scripty". I'll change it because I like you, Jon.

Reply:
Your wish is my command good ser